Cargo.toml

```diff
@@ -19,6 +19,7 @@ anyhow = { workspace = true }
 async-trait = { workspace = true }
 bytes = { workspace = true }
 clap = { version = "3.2.24", features = ["derive", "env"] }
+clap-markdown = { git = "https://github.com/itowlson/clap-markdown", branch = "clap-v3-compat" }
```

Comment:
Should we move the fork to spinframework's repo?

Reply:
I am not sure what to do with the fork.  It can't live in my personal account I agree.  But it feels awkward to have something so disconnected from Spin and so specifically about Clap in spinframework.  Maybe that's just me.  We could make it a crate in the Spin project, but then we would lose traceability to the original repo and author, which feels rude.

Anyway I'm happy to follow whatever you and other folks more steeped on open-source etiquette advise!